lib/artifactory/ArtifactoryLayerService.js

```diff
@@ -0,0 +1,129 @@
+"use strict";
```

Comment:
no need to upload lib dir, no?
it's compiled by src

Reply:
Actually, as I wrote above, it's a weird "compilation" that the original plugin does we need to push the built files to our repository in order to keep using this plugin. We can consider refactoring this part and don't upload the /lib folder as expected, but I think this is not part of this task's scope.

I tried to remove the lib folder with some changes, and ran a lot of pipeline in Jenkins, but all failed for "Serverless plugin "serverless-layers" not found. Make sure it's installed and listed in the "plugins" section of your serverless config file. Run "serverless plugin install -n serverless-layers" to install it."
Only after adding this lib folder again, the pipeline succeeded.
Also, the last PRs for this plugin after forking it from the original repository also included the lib folder + the changes...
So we should consider syncing the original plugin to our forked repository, but again - I don't think this is this task's scope...